shared/chat/conversation/input-area/normal/platform-input.desktop.js

```diff
@@ -23,17 +26,19 @@ const MentionCatcher = ({onClick}) => (
 
 type State = {
   emojiPickerOpen: boolean,
+  explodingPopupOpen: boolean,
   hasText: boolean,
 }
 
-class PlatformInput extends Component<PlatformInputProps, State> {
+class PlatformInput extends Component<PlatformInputProps & FloatingMenuParentProps, State> {
   _input: ?Input
   _fileInput: ?HTMLInputElement
 
-  constructor(props: PlatformInputProps) {
+  constructor(props: PlatformInputProps & FloatingMenuParentProps) {
     super(props)
     this.state = {
       emojiPickerOpen: false,
+      explodingPopupOpen: false,
```

Comment:
This comes in through the `FloatingMenuParentHOC` too, as `props.showingMenu` and `props.toggleShowingMenu()`